Makefile

```diff
@@ -63,20 +67,20 @@ scripts-permission:
 depends: $(BUILD_DIR)/ruby $(BUILD_DIR)/node
 
 # update dependencies information
-update: depends
+update: clean depends
```

Comment:
`make update` is a utility function to update to the latest versions of dependencies.  When it is run, we should clean as well to ensure build output includes the updated dependencies.